services/static-webserver/client/source/class/osparc/data/model/NodeProgressSequence.js

```diff
@@ -311,44 +375,54 @@ qx.Class.define("osparc.data.model.NodeProgressSequence", {
     },
 
     __applySidecarPulling: function(value) {
-      if (value > 0) {
-        this.setClusterUpScaling(1);
+      if (value.value > 0) {
+        const defaultEndVals = this.getDefaultEndValues();
+        this.setClusterUpScaling(defaultEndVals);
+        this.self().updateProgressLabel(this.__clusterUpScalingLayout, defaultEndVals);
```

Comment:
```suggestion
        this.setClusterUpScaling(defaultEndVals);
```
There is no need to call ``this.self().updateProgressLabel(this.__clusterUpScalingLayout, defaultEndVals);`` it will be called in the ``__applyClusterUpScaling``. Same goes for the other steps.